tools/setup-envtest/README.md

```diff
@@ -8,7 +8,7 @@ To use it, just go-install it with Golang 1.24+ (it's a separate, self-contained
 module):
 
 ```shell
-go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
+go install sigs.k8s.io/controller-runtime/tools/setup-envtest@release-0.22
```

Comment:
Maybe it would be better to instead tell ppl to download it from a github release to avoid keeping this up to date? We will most likely forget to do that